bazel/ray.bzl

```diff
@@ -103,3 +103,41 @@ def define_java_module(
             "{auto_gen_header}": "<!-- This file is auto-generated by Bazel from pom_template.xml, do not modify it. -->",
         },
     )
+
+def copy_to_workspace(name, srcs, subdir = ""):
```

Comment:
subdir -> dstdir? i didn't get what subdir does until reading the source of this.